lib/Runtime/Library/JavascriptProxy.cpp

```diff
@@ -645,6 +645,21 @@ namespace Js
         }
         else
         {
+            // ES2017 Spec'ed (9.1.9.1): 
+            // If existingDescriptor is not undefined, then
+            //    If IsAccessorDescriptor(existingDescriptor) is true, return false.
+            //    If existingDescriptor.[[Writable]] is false, return false.
+
+            if (proxyPropertyDescriptor.IsAccessorDescriptor())
+            {
+                return FALSE;
+            }
+
+            if (proxyPropertyDescriptor.WritableSpecified() && !proxyPropertyDescriptor.IsWritable())
+            {
+                return FALSE;
+            }
+
```

Comment:
That is the test case added - the handler['defineProperty'] was called before my fix. After my fix it is not called. Is that what you are looking for?

---
In reply to: [95928059](https://github.com/Microsoft/ChakraCore/pull/2374#discussion_r95928059) [](ancestors = 95928059)